src/renderer/components/archive/entry.js

```diff
@@ -123,7 +122,7 @@ class Entry extends PureComponent {
         <EmptyView
           caption={t('entry.select-or-create-an-entry')}
           className={styles.emptyView}
-          imageSrc={bench}
+          imageSrc={getRandomIllustration()}
```

Comment:
Would this be safer setting it to state? If a re-render occurs for any reason, the image will change, right?

Reply:
@perry-mitchell actually this was intentional, so for example if the user sees this view again, the image will change to a new one. I have some mac apps that do this and it's really nice. so you don't have to quit the app to see a new one. what do you think?

Reply:
I'm happy with it 😊